api/api/src/test/scala/uk/ac/wellcome/platform/api/services/ElasticsearchServiceTest.scala

```diff
@@ -229,6 +240,147 @@ class ElasticsearchServiceTest
     }
   }
 
+  describe("searches using JustBoostQuery") {
+    it("finds results for a JustBoostQuery search") {
+      withLocalWorksIndex { index =>
+        // Longer text used to ensure signal in TF/IDF
+        val text = "Text that contains Aegean"
+
+        val work = createIdentifiedWorkWith()
+        val matchingTitle =
+          createIdentifiedWorkFrom(work)(title = f"$text title")
+        val exactMatchingTitle =
+          createIdentifiedWorkFrom(work)(title = f"Aegean title")
+        val matchingSubject = createIdentifiedWorkFrom(work)(
+          subjects = List(createSubjectWith(f"$text subject")))
+        val matchingGenre = createIdentifiedWorkFrom(work)(
+          genres = List(createGenreWith(f"$text genre")))
+        val matchingDescription = createIdentifiedWorkFrom(work)(
+          description = Some(f"$text description"))
+        val noMatch = createIdentifiedWorkFrom(work)(title = "Before a Bengal")
+
+        insertIntoElasticsearch(
+          index,
+          matchingTitle,
+          noMatch,
+          matchingSubject,
+          exactMatchingTitle,
+          matchingDescription,
+          matchingGenre)
+
+        val results =
+          searchResults(index = index, workQuery = JustBoostQuery("Aegean"))
+
+        results should have length 5
+
+        results.head shouldBe exactMatchingTitle
+        results.slice(1, 3) should contain theSameElementsAs List(
+          matchingSubject,
+          matchingGenre)
+        results(3) shouldBe matchingTitle
+        results(4) shouldBe matchingDescription
+      }
+    }
+  }
+
+  describe("searches using BroaderBoostQuery") {
+    it("finds results for a BroaderBoostQuery search") {
+      withLocalWorksIndex { index =>
+        // Longer text used to ensure signal in TF/IDF
+        val text = "Text that contains Aegean"
+
+        val work = createIdentifiedWorkWith()
+        val matchingTitle =
+          createIdentifiedWorkFrom(work)(title = f"$text title")
+        val matchingSubject = createIdentifiedWorkFrom(work)(
+          subjects = List(createSubjectWith(f"$text subject")))
+        val matchingGenre = createIdentifiedWorkFrom(work)(
+          genres = List(createGenreWith(f"$text genre")))
+        val matchingDescription = createIdentifiedWorkFrom(work)(
+          description = Some(f"$text description"))
+        val matchingLettering =
+          createIdentifiedWorkFrom(work)(lettering = Some(f"$text lettering"))
+        val matchingContributor =
+          createIdentifiedWorkFrom(work)(
+            contributors =
+              List(createPersonContributorWith(f"$text contributor")))
+        val noMatch = createIdentifiedWorkFrom(work)(title = "Before a Bengal")
```

Comment:
As these are repeated, I’d be inclined to pull them out of the test bodies and define them as constants on the test class/inside a `describe` block. Reduce the boilerplate.